kbase-extension/static/kbase/js/widgets/appWidgets2/validation.js

```diff
@@ -43,12 +55,66 @@ define([
             };
         }
 
+        /**
+         * Validates whether the value is a valid object data palette reference path,
+         * consisting of only numerical values in the reference (e.g.: 1/2/3;4/5/6;7/8/9)
+         * Note that this doesn't check to see if the path is real, just if it's syntactically
+         * valid.
+         * @param {string} value
+         * @param {Object} options
+         * - required - boolean
+         * @returns {Object}
+         */
+        function validateWorkspaceDataPaletteRef(value, options) {
```

Comment:
I don't **think** this gets used anymore, but it seemed safer to preserve it for now.